apitrackingmore.py

```diff
@@ -11,19 +11,26 @@
 key = config['TRACKINGMORE']['key']
 trackingmore.set_api_key(key)
 
+carriers = ('cainiao', 'laos-post', 'dhl', 'sunyou')
+
 def get(code, times):
-    try:
-        td = trackingmore.create_tracking_data('cainiao', code)
-        trackingmore.create_tracking_item(td)
-    except trackingmore.trackingmore.TrackingMoreAPIException as e:
-        if e.err_code == 4019 or e.err_code == 4021:
-            return status.OFFLINE
-        if e.err_code == 4016: # Already exists
-            try:
-                td = trackingmore.get_tracking_item('cainiao', code)
-            except trackingmore.trackingmore.TrackingMoreAPIException as e:
-                if e.err_code == 4019 or e.err_code == 4021:
-                    return status.OFFLINE
+    td = None
+    for carrier in carriers:
+        try:
+            td = trackingmore.create_tracking_data(carrier, code)
+            trackingmore.create_tracking_item(td)
+        except trackingmore.trackingmore.TrackingMoreAPIException as e:
+            if e.err_code == 4019 or e.err_code == 4021:
+                return status.OFFLINE
+            if e.err_code == 4016: # Already exists
+                try:
+                    td = trackingmore.get_tracking_item(carrier, code)
+                except trackingmore.trackingmore.TrackingMoreAPIException as e:
+                    if e.err_code == 4019 or e.err_code == 4021:
+                        return status.OFFLINE
```

Comment:
Muito obrigado pelo PR @Charliiee!

Esse trecho ta um pouco confuso (já estava antes da sua mudança) e os 6 níveis de identação não ajudam :/ Minha sugestão aqui seria criar uma função separada pra procurar e/ou criar o tracking item, algo como: `get_or_create_tracking_item(carrier, code)`. Com essa função, poderíamos fazer algo assim:

```python
carriers = ('cainiao', 'laos-post', 'dhl', 'sunyou')
for carrier in carriers:
    status, tracking_data = get_or_create_tracking_item(carrier, code)

    if status in (4019, 4021):
        return status.OFFLINE

    if tracking_data and tracking_data["status"] != "notfound" and len(tracking_data) > 10:
        break

else:
    return status.NOT_FOUND

return formato_obj(tracking_data)
```

O código dentro do bloco `else` só será executado se o `break` dentro do loop **não** for executado.

Para isso funcionar, a função `get_or_create_tracking_item` teria que retornar sempre o status da requisição e um dicionário, seja ele com conteúdo esperado, ou vazio se alguma exceção ocorrer.

O que você acha? Faz sentido?